beta-src/src/components/ui/WDCountryTable.tsx

```diff
@@ -0,0 +1,176 @@
+  const WDTableCell = styled(TableCell)(() => {
+    const padding = mobileLandscapeLayout ? 6 : "6px 16px";
+    return {
+      [`&.${tableCellClasses.head}`]: {
+        borderBottom: 0,
+        lineHeight: "normal",
+        padding,
+      },
+      [`&.${tableCellClasses.body}`]: {
+        borderBottom: 0,
+        padding,
+      },
+    };
+  });
+  return (
+    <div>
+      <Table aria-label="country info table" size="small" stickyHeader>
+        <TableHead sx={{ height: "55px" }}>
+          <TableRow sx={{ verticalAlign: "top" }}>
+            {columns.map((column) => (
+              <WDTableCell key={column.id} align={column.align}>
+                {column.icon && <column.icon />}
+                <Box sx={{ fontSize: 10, fontWeight: 400 }}>
+                  {column.label.toUpperCase()}
+                </Box>
+              </WDTableCell>
+            ))}
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {countries.map((country) => (
+            <React.Fragment key={country.power}>
+              <TableRow>
+                {columns.map((column) => {
+                  const style = {
+                    color: theme.palette.primary.main,
+                    fontWeight: 400,
+                  };
+                  let value: string;
+                  switch (column.id) {
+                    case "power":
+                      value = mobileLandscapeLayout
+                        ? country.abbr.toUpperCase()
+                        : country.power.toUpperCase();
+                      style.color = country.color;
+                      style.fontWeight = 700;
+                      break;
+                    case "delaysLeft":
+                      value = `${country[column.id]}/${maxDelays}`;
+                      break;
+                    default:
+                      value = `${country[column.id]}`;
+                      break;
+                  }
+                  return (
+                    <WDTableCell key={column.id} align={column.align}>
+                      <span style={style}>{value}</span>
+                    </WDTableCell>
+                  );
+                })}
+              </TableRow>
+              {country.votes && (
+                <TableRow>
+                  <WDTableCell
+                    sx={{
+                      fontSize: "70%",
+                      paddingTop: "0px !important",
+                      fontWeight: 700,
+                    }}
+                    colSpan={columns.length}
+                  >
+                    <Box
+                      sx={{
+                        color: theme.palette.action.disabledBackground,
+                        display: "inline-block",
+                        marginRight: 1.5,
+                      }}
+                    >
+                      VOTED
+                    </Box>
+                    {Object.entries(country.votes).map(
+                      (data) =>
```

Comment:
it would be nice to have a more descriptive word than data...i guess it would be vote, but then it's a key value pair...so maybe instead of `(data)` we have `([key, value])` it's just weird seeing data[0] and data[1] because at first glance i don't really know what this data is

Reply:
Yea, I thought about changing it to `voteType` and `didVote` but the code became slightly longer by about 6 lines so I opted to go for less verbose code. I think the 6 lines for better clarity is a good tradeoff though, I'll make this update once I have some free time.